cli.go

```diff
@@ -52,6 +57,7 @@ func (app *App) NewPlanCommand() *cli.Command {
 				Option{
 					Repository: c.String("repository"),
 					NoColor:    c.Bool("no-color"),
+					Format:     newOutputFormatFrom(c.String("format")),
```

Comment:
`newOutputFormatFrom` should return `(outputFormat, error)`.
Because the `unknown output format` error raised at this point is better than the output phase.

Reply:
do early return in `newOutputFormatFrom` https://github.com/fujiwara/ecrm/pull/8/commits/cfce6d9abe96abfd22222ba3bfb9dde454b51e48